docs/_posts/2018-02-22-units-of-measurement.md

```diff
@@ -0,0 +1,118 @@
+Up to now these units where not represented in Eclipse SmartHome and the user had to "know" its units and model items and rules accordingly.
+In order to unify the different values and units and let the smart home make use of it, Eclipse SmartHome now comes with support for "Units of Measurement" or in short "UoM".
+
+##### Units of Measurement
+From the [UoM Wikipedia](https://en.wikipedia.org/wiki/Units_of_measurement) article: "A unit of measurement is a definite magnitude of a quantity, defined and adopted by convention or by law, that is used as a standard for measurement of the same kind of quantity. Any other quantity of that kind can be expressed as a multiple of the unit of measurement."
+Units are organised in systems of units and the global standard for units is the [International System of Units (SI)](https://en.wikipedia.org/wiki/International_System_of_Units).
+Apart from SI there are other systems in use like the imperial unit system. The same physical quantity can be expressed in different units and mathematical transformations exist to convert between them.
+Depending on the users locale or cultural preference a specific unit might fit the users needs. For temperature it is common to use degree Celsius (°C) in most places around the globe but degree Fahrenheit (°F) is used in the United States of America.
+
+##### UoM data with QuantityType
+Sensor devices and internet services connected to Eclipse SmartHome as Things provide their scalar values in an implicit unit of measurement. The binding developer knows what exact unit the sensor or service data uses and can pass those values to the framework using the new [`QuantityType`](../documentation/concepts/units-of-measurement.html). QuantityType represents the scalar value with the corresponding unit. It also serves as the main API to convert between different units.
+The Yahoo Weather binding makes use of `QuantityType` and publishes state updates in the following form:
+
+```(java)
+QuantityType<Temperature> state = new QuantityType<>(tempDouble, SIUnits.CELSIUS);
+updateState(temperatureChannelUID, state);
+```
+
+For this example the binding developer knows from the Yahoo API description that degree Celsius is used for temperature values. For pressure values the API description states that Hecto Pascal (hPa) is used, so a state update for the pressure channel looks like this:
+
+```(java)
+QuantityType<Pressure> state = new QuantityType<>(pressDouble, HECTO(SIUnits.PASCAL));
+updateState(pressureChannelUID, state);
+```
+
+In the temperature example the state is marked to be a "<Temperature>" typed QuantitType. It will only accept units which correspond to the dimension type, like degree Celsius, degree Fahrenheit or kelvin.
```

Comment:
Does `"<Temperature>"` work for you?
At least Github's Markdown parser cannot display it correctly.

Reply:
good find, thanks @maggu2810.